### PR TITLE
[fix] Fix error problem on h2 startup data quality rule management page

### PR DESCRIPTION
**Pull request by boy-xiaozhang**
## Purpose of the pull request

Fix error problem on h2 startup data quality rule management page

## Brief change log

add h2 database keyword conflict resolve

## Verify this pull request

add UT to [dolphinscheduler-dao]

join issue #12107

### CODE REVIEW

File: dolphinscheduler-dao/src/main/resources/org/apache/dolphinscheduler/dao/mapper/DqRuleInputEntryMapper.xml [hunk 1]
```diff
@@ -24,7 +24,7 @@
                a.field,
                a.type,
                a.title,
-               a.value,
+               a.`value`,
```
Comment (ruanwenjun) [inline dolphinscheduler-dao/src/main/resources/org/apache/dolphinscheduler/dao/mapper/DqRuleInputEntryMapper.xml:27]: Could you please add UT for this? we have h2 in our UT.

Comment (boy-xiaozhang) [reply dolphinscheduler-dao/src/main/resources/org/apache/dolphinscheduler/dao/mapper/DqRuleInputEntryMapper.xml:27]: ok,sure

**Review by caishunfeng — APPROVED** [review]
LGTM

**Review by ruanwenjun — APPROVED** [review]
LGTM

**Pull request merged**